### PR TITLE
add CONTRIBUTING.md

### LINKED ISSUE #56: Add CONTRIBUTING.md
Add a list of steps to correctly open a PR.

### PR DESCRIPTION
**Pull request by altafan**
Closes #56

### CODE REVIEW

**Comment by tiero** [comment]
I would add more information on the process of review, master-read only, mandatory review and one test suite.

**Comment by lorenzodisidoro** [comment]
We could split the file in more paragraphs, something like this:
- **Contributing** (modus operandi for opening PRs) where we explain contributing workflow 
- **Rules** eg. explain opened PR into the comment, linking the related issues, squashing, ...
- **Categories** define the boundaries of main PR categories because cannot be mixed (eg. bug fixing, new feature, refactoring, ...)

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# Contributing
+
+Hereafter is detailed how to proceed in order to contribute to Nigiri.
+
+## Rules
```
Comment (tiero) [inline CONTRIBUTING.md]: make bold on the keywords (master branch, Pull requests etc..)

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
+# Contributing
```
Comment (tiero) [inline CONTRIBUTING.md]: Link this in the readme.md

**Comment by tiero** [comment]
I would keep in hold, meanwhile we find a solution for import paths

**Pull request closed without merging**